source/fundamentals/indexes.txt

```diff
@@ -242,9 +244,20 @@ language as an option when creating the index.
    Text indexes differ from the more powerful :atlas:`Atlas full text search indexes </atlas-search/>` Atlas users
    should use Atlas search.
 
-.. warning::
+.. warning:: Restriction on one text index per collection. 
+
+   If you want to create a text index for multiple text fields, you need
+   to create an compound index to include them. 
 
-   Collections are limited to one text index. Keep in mind that a text index can cover multiple fields.
+   The following creates a compound text index for the ``title`` and ``genre``
+   fields:
+
+   .. code-block:: java
+   
+      collection.createIndex(Indexes.compoundIndex(Indexes.text("title"), Indexes.text("genre")));
+   
+   For more information on compound text index restrictions, see the
+   server manual entry on :manual:`Compound Text Indexes <core/index-text/#std-label-text-index-compound>`.
```

Comment:
For consistency, should we capitalize Server Manual Entry or keep it lowercase across the docs? I'm open to either, but the former (proper noun form) might help the user identify that we're referring to something specific.